### ISSUE TITLE
Should auxiliary affordance outputs be included in RT-DETR decoder?

### ISSUE THREAD

**Issue by anh0001** [anh0001/affomernet#14]
Currently, the RT-DETR decoder implementation (`rtdetr_decoder.py`) includes auxiliary outputs for object detection (bounding boxes and class logits) but doesn't provide auxiliary outputs for affordances. 

The current auxiliary output structure in `_set_aux_loss()`:
```python
def _set_aux_loss(self, outputs_class, outputs_coord):
    return [{'pred_logits': a, 'pred_boxes': b}
            for a, b in zip(outputs_class, outputs_coord)]
```

Since affordances are also being predicted and we maintain auxiliary losses for detection outputs for better training, should we also include affordance outputs in the auxiliary loss structure? This would be consistent with the multi-task training paradigm.

Potential changes needed:
1. Modify decoder output structure to include affordance features at each decoder layer
2. Update `_set_aux_loss()` to include affordance predictions
3. Ensure the affordance branch can handle auxiliary outputs for training

This would allow for deeper supervision of the affordance prediction task across decoder layers, potentially improving affordance learning.

Would love to hear thoughts on whether this addition would be beneficial for training stability and model performance.
